jetty-ee11/jetty-ee11-jaspi/src/main/java/org/eclipse/jetty/ee11/security/jaspi/DefaultAuthConfigFactory.java

```diff
@@ -153,16 +143,9 @@ public RegistrationContext getRegistrationContext(String registrationID)
     @Override
     public void refresh()
     {
-        checkPermission();
-
         // TODO: maybe we should re-construct providers created from classname.
     }
 
```

Comment:
Having permission checks in EE10 but not in EE11 looks fishy, but could be correct. Maybe we should add a comment explaining why EE11 doesn't do that?

Reply:
@lachlan-roberts  can you look at this one?